106-auto-rebalance-on-imbalanced-clusters.md

```diff
@@ -0,0 +1,520 @@
+# Auto-rebalance on imbalanced clusters
+
+This proposal introduces support for automatic partition rebalancing whenever a Kafka cluster becomes imbalanced due to unevenly distributed replicas, overloaded brokers, or similar issues.
+This feature is opt-in, and when enabled, allows the Strimzi Operator to automatically execute partition rebalances through `KafkaRebalance` custom resources whenever partition imbalances are detected by Cruise Control.
+
+## Motivation
+
+Currently, if the cluster is imbalanced, the user would need to manually rebalance the cluster by using the `KafkaRebalance` custom resource.
+With smaller clusters, it is feasible to fix things manually. However, for larger ones it can be very time-consuming, or just not feasible, to fix all the imbalances on your own.
```

Comment:
What is `very time-consuming` about this for larger clusters? Isn't the process exactly the same for small and large cluster when you use `KafkaRebalance`? It just takes longer between the steps. But that does not mean the process itself is more time-consuming.